lvsfunc/comparison.py

```diff
@@ -605,8 +609,13 @@ def diff(*clips: vs.VideoNode,
     if (clips and len(clips) != 2) or (namedclips and len(namedclips) != 2):
         raise CustomValueError("Must pass exactly 2 `clips` or `namedclips`!", diff)
 
-    if not 1 <= thr < 128:
-        raise CustomValueError(f"`thr` must be between 1 and 128, not {thr}!", diff)
+    if not thr < 128:
```

Comment:
Why did you remove the positive number check?

Reply:
thr was designed to be able to be negative. so this is fixing a bug,

Reply:
Then check if abs(thr) <= 128 or ths is in [-128, 128]

Reply:
Oh, I'm sorry, I got it wrong. It should check for 0 <= thr < 128 (or perhaps 0 < thr < 128). I will check and fix it.

Reply:
Gotcha, thanks.
You can resolve the convo after committing.